source/test/F0.Minesweeper.Components.Tests/Pages/Game/Modules/GameEndStatusTests.razor

```diff
@@ -75,4 +62,36 @@
 		// Assert
 		componentUnderTest.MarkupMatches(expectedMarkup);
 	}
+
+	[Fact]
+	public void OnRestartGame_GameEndTextIsHidden()
+	{
+		// Arrange
+		RenderFragment expectedMarkup = @<div id='f0-gameendstatus'><p class='f0-end-text-invisible' /></div>
+	;
```

Comment:
Did you intend to have the semicolon on the next line?

Reply:
Dear Microsoft. Please fix formatting in razor files. According to https://github.com/dotnet/razor-tooling/issues/4807 this could be fixed with the full release :) 

I fixed all formatting issues in the file :)